File: Examenw/src/main/java/com/pe/edu/upeu/examenw/entity/Muebles.java
package com.pe.edu.upeu.examenw.entity;

import java.time.LocalDate;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Entity
@Data
public class Muebles {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String nombre;
    private String descripcion;
    private String tipo;
    private double precio;
    private int stock;
    private String dimensiones;
    private String color;
    private String material;
}

File: Proyecto/app/src/main/java/pe/edu/upeu/proyecto/Muebles.kt
package pe.edu.upeu.proyecto

import java.math.BigDecimal

data class Muebles(
    val id: Long,
    val nombre: String,
    val descripcion: String,
    val tipo: String,
    val precio: BigDecimal,
    val stock: Int,
    val dimensiones: String,
    val color: String,
    val material: String,
)
